Apps/Sandcastle/gallery/Web Map Tile Service with Time.html

```diff
@@ -0,0 +1,95 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <meta name="description" content="Use Viewer to start building new applications or easily embed Cesium into existing applications.">
+    <meta name="cesium-sandcastle-labels" content="Beginner, Showcases">
+    <title>Cesium Demo</title>
+    <script type="text/javascript" src="../Sandcastle-header.js"></script>
+    <script type="text/javascript" src="../../../ThirdParty/requirejs-2.1.20/require.js"></script>
+    <script type="text/javascript">
+    require.config({
+        baseUrl : '../../../Source',
+        waitSeconds : 60
+    });
+    </script>
+</head>
+<body class="sandcastle-loading" data-sandcastle-bucket="bucket-requirejs.html">
+<style>
+    @import url(../templates/bucket.css);
+</style>
+<div id="cesiumContainer" class="fullSize"></div>
+<div id="loadingOverlay"><h1>Loading...</h1></div>
+<div id="toolbar"></div>
+<script id="cesium_sandcastle_script">
+function startup(Cesium) {
+    'use strict';
+//Sandcastle_Begin
+var viewer = new Cesium.Viewer('cesiumContainer');
+
+function dataCallback(interval, index) {
+    var time;
+    if (index === 0) { // leading
+        time = Cesium.JulianDate.toIso8601(interval.stop);
+    } else {
+        time = Cesium.JulianDate.toIso8601(interval.start);
+    }
+
+    return {
+        Time: time
+    };
+}
+
+var times1 = Cesium.TimeIntervalCollection.fromIso8601({
+    iso8601: '2015-07-30/2017-06-16/P1D',
+    leadingInterval: true,
+    trailingInterval: true,
+    isStopIncluded: false, // We want stop time to be part of the trailing interval
+    dataCallback: dataCallback
+});
+
+var times2 = Cesium.TimeIntervalCollection.fromIso8601DateArray({
+    iso8601Dates: ['2015-07-30', '2015-07-31', '2015-08-01', '2015-08-02', '2015-08-03', '2015-08-04', '2015-08-05'],
+    leadingInterval: true,
+    trailingInterval: true,
+    isStopIncluded: false, // We want stop time to be part of the trailing interval
+    dataCallback: dataCallback
+});
+
+var times3 = Cesium.TimeIntervalCollection.fromIso8601DurationArray({
+    epoch: Cesium.JulianDate.fromIso8601('2015-07-30'),
+    iso8601Durations: ['P0D', 'P1D', 'P2D', 'P3D', 'P4D', 'P5D', 'P6D'],
+    leadingInterval: true,
+    trailingInterval: true,
+    isStopIncluded: false, // We want stop time to be part of the trailing interval
+    dataCallback: dataCallback
+});
+
+// Add a WMTS imagery layer
+var imageryLayers = viewer.imageryLayers;
+imageryLayers.addImageryProvider(new Cesium.WebMapTileServiceImageryProvider({
+    url : 'https://gibs.earthdata.nasa.gov/wmts/epsg4326/best/AMSR2_Snow_Water_Equivalent/default/{Time}/{TileMatrixSet}/{TileMatrix}/{TileRow}/{TileCol}.png',
+    layer : 'AMSR2_Snow_Water_Equivalent',
+    style : 'default',
+    tileMatrixSetID : '2km',
+    maximumLevel : 5,
+    format : 'image/png',
+    clock: viewer.clock,
+    times: times1
+}));
+
+viewer.clock.multiplier = 14400;
```

Comment:
Perhaps make this quite a bit faster so it is almost as fast as a video - it shows better and seems to run fine.